types/jquery-monthpicker/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for non-npm package MonthPicker 3.0
+// Project: <https://github.com/KidSysco/jquery-ui-month-picker>
```

Comment:
Project link is incorrect from npm package: https://www.npmjs.com/package/monthpicker
It needs to be https://github.com/ddiebel/monthpicker

Reply:
I'm not sure I understand why I should use a different github repository?
The module in https://github.com/ddiebel/monthpicker is a different repository.

Reply:
Sorry i read that as npm package and hence got confused since the npm pacakge for that points to different repo

Can you name it jquery-monthPicker in the header instead?